utils/src/main/java/io/atomix/utils/misc/TestUtil.java

```diff
@@ -0,0 +1,5 @@
+package io.atomix.utils.misc;
+
+public class TestUtil {
```

Comment:
?

Reply:
Sorry was a mistake i removed it already